swan-accpy/Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG VERSION_PARENT=v0.0.15
+
+FROM gitlab-registry.cern.ch/swan/docker-images/jupyter/swan-cern:${VERSION_PARENT}
+
+LABEL maintainer="swan-admins@cern.ch"
+ARG NB_UID="1000"
+ARG BUILD_TAG=daily
+ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
+ENV ACCPY_PATH="/opt/acc-py"
+
+RUN echo "Building swan-accpy image with tag ${VERSION_DOCKER_IMAGE} from parent tag ${VERSION_PARENT}."
+
+USER root
+
+RUN for version in "2020.11" "2021.12" "2023.06"; do \
+        wget http://bewww.cern.ch/ap/acc-py/installers/acc-py-${version}-installer.sh && \
+        chmod +x acc-py-${version}-installer.sh && \
+        ./acc-py-${version}-installer.sh --installation-root ${ACCPY_PATH} && \
+        rm -f acc-py-${version}-installer.sh && \
+        python -m pip install ipykernel && \
+        source ${ACCPY_PATH}/base/${version}/setup.sh && \
+        ${ACCPY_PATH}/base/${version}/bin/python -m ensurepip --upgrade && \
+        ${ACCPY_PATH}/base/${version}/bin/pip3 install ipykernel && \
+        deactivate; \
+    done
+
+RUN dnf install -y \
+    java-11-openjdk.x86_64 && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+# Eventually, this need to be moved to swan-cern
+COPY SwanCustomEnvironments /tmp/SwanCustomEnvironments
+RUN pip install /tmp/SwanCustomEnvironments && \
+    rm -rf /tmp/SwanCustomEnvironments
```

Comment:
This would be better done via installation from PyPi.

For this, what we could do is already merge `SwanCustomEnvironments` into master in jupyter-extensions, since having one more extension there would not impact any other extension.